README.adoc

```diff
@@ -172,8 +189,13 @@ and whether to create the database if it does not already exist.
 
 == Performing CRUD operations using JPA
 
-CRUD is an abbreviation that defines the four basic functionalities that can be performed on a 
-database: create, read, update, and delete.
+Every entity object has a lifecycle that consists of four states: new, managed, detached and removed. 
```

Comment:
Removed really isn't a "state", when you call em.remove(), that entity is marked for removal. It's still managed until you punt it with em.clear() or em.detach() or complete the transaction.